src/main/resources/assets/gregtech/lang/en_us.lang

```diff
@@ -706,13 +706,13 @@ cover.conveyor.title=Conveyor Cover Settings (%s)
 cover.conveyor.transfer_rate=%s items/sec
 cover.conveyor.mode.export=Mode: Export
 cover.conveyor.mode.import=Mode: Import
+cover.universal.manual_import_export.mode.disabled=Disallow Manual I/O
+cover.universal.manual_import_export.mode.filtered=Filtered Manual I/O
+cover.universal.manual_import_export.mode.unfiltered=Unfiltered Manual I/O
+cover.universal.manual_import_export.mode.description=§eDisallow§r - in this mode, items/fluids will only move as specified by the cover and its filter. /n§eAllow Filtered§r - Items/fluids can be extracted and inserted indepedently of the cover mode, as long as its match filter (if any). /n§eAllow Unfiltered§r - In this mode, items/fluids can be moved indepedently of the cover mode. Filter applies to the items inserted or extracted by this cover
```

Comment:
`§eDisallow§r - Items/fluids will only move as specified by the cover and its filter. /n§eAllow Filtered§r - Items/fluids can be extracted and inserted independently of the cover mode, as long as its filter matches (if any). /n§eAllow Unfiltered§r - Items/fluids can be moved independently of the cover mode. Filter applies to the items inserted or extracted by this cover`